xsd-core/src/main/java/module-info.java

```diff
@@ -26,4 +26,7 @@
     exports io.ballerina.xsd.core.visitor;
     exports io.ballerina.xsd.core.component;
     exports io.ballerina.xsd.core.diagnostic;
+    exports io.ballerina.xsd.core.context;
+    exports io.ballerina.xsd.core.response;
+    exports io.ballerina.xsd.core.node;
```

Comment:
We dont needs to export this

Reply:
Removed the `io.ballerina.xsd.core.node;` [here](
f1a6ca7) 
But we may need `io.ballerina.xsd.core.response;` as it is added because the `Response` class is moved into a separate package